datasets/nq_open/README.md

```diff
@@ -0,0 +1,203 @@
+- **Repository:** https://github.com/google-research-datasets/natural-questions/tree/master/nq_open
+- **Paper:** https://www.aclweb.org/anthology/P19-1612.pdf
+- **Leaderboard:** https://ai.google.com/research/NaturalQuestions/efficientqa
+- **Point of Contact:** efficientqa@googlegroups.com
+
+### Dataset Summary
+
+The NQ-Open task, introduced by Lee et.al. 2019,
+is an open domain question answering benchmark that is derived from Natural Questions.
+The goal is to predict an English answer string for an input English question.
+All questions can be answered using the contents of English Wikipedia.
+
+### Supported Tasks and Leaderboards
+
+Open Domain Question-Answering, 
+EfficientQA Leaderboard: https://ai.google.com/research/NaturalQuestions/efficientqa
+
+### Languages
+
+English (`en`)
+
+## Dataset Structure
+
+### Data Instances
+
+{
+    "question": "names of the metropolitan municipalities in south africa",
+    "answer": [
+        "Mangaung Metropolitan Municipality",
+        "Nelson Mandela Bay Metropolitan Municipality",
+        "eThekwini Metropolitan Municipality",
+        "City of Tshwane Metropolitan Municipality",
+        "City of Johannesburg Metropolitan Municipality",
+        "Buffalo City Metropolitan Municipality",
+        "City of Ekurhuleni Metropolitan Municipality"
+    ]
+}
+
+### Data Fields
+
+- `question` - Input open domain question.
+- `answer` - List of possible answers to the question
+
+### Data Splits
+
+- Train : 87925
+- validation : 1800
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+Natural Questions contains question from aggregated queries to Google Search (Kwiatkowski et al., 2019). To gather an open version of this dataset, we only keep questions with short answers and discard the given evidence document. Answers with many tokens often resemble extractive snippets rather than canonical answers, so we discard answers with more than 5 tokens.
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+Evaluating on this diverse set of question-answer
+pairs is crucial, because all existing datasets have
+inherent biases that are problematic for open domain QA systems with learned retrieval. 
+In the Natural Questions dataset the question askers do not already know the answer. This accurately reflects a distribution of genuine information-seeking questions.
+However, annotators must separately find correct
+answers, which requires assistance from automatic
+tools and can introduce a moderate bias towards
+results from the tool.
```

Comment:
```suggestion
Evaluating on this diverse set of question-answer pairs is crucial, because all existing datasets have inherent biases that are problematic for open domain QA systems with learned retrieval. 
In the Natural Questions dataset the question askers do not already know the answer. This accurately reflects a distribution of genuine information-seeking questions.
However, annotators must separately find correct answers, which requires assistance from automatic tools and can introduce a moderate bias towards results from the tool.
```